tensorflow_serving/util/class_registration.h

```diff
@@ -266,7 +266,10 @@ class ClassRegistry {
         protobuf::MessageFactory::generated_factory()
             ->GetPrototype(descriptor)
             ->New());
-    any_config.UnpackTo(config.get());
+    if (!any_config.UnpackTo(config.get())) {
+      return errors::InvalidArgument("Malformed content of Any.value: ",
```

Comment:
How do we know that false means there was a problem with the .value component specifically?

Maybe s/Malformed content of Any.value/Malformed content of Any/

Reply:
I'd say we know: Any only has two fields, `type_id` and `value`. `type_id` is resolved and the corresponding proto descriptor is loaded at this point.

Reply:
Well, the internal representation of the Any could, in principle, change over time. But it's not a big deal either way.

Reply:
Good point, I changed the error message.